public/locales/en/sheet.json

```diff
@@ -37,6 +37,8 @@
     "atk": "ATK Increase"
   },
   "skillCDRed": "Ele. Skill CD Reduction",
+  "pressCD": "Press CD",
+  "holdCD": "Hold CD",
```

Comment:
So these values are already piped in from pipeline using genshin's translation.
![image](https://user-images.githubusercontent.com/1754901/156920378-13355b39-42e4-4819-870c-304d55cd8fe6.png)

The values in `sheet.json` needs to be manually translated for every language. The `sheet_gen.json` are values we extract from the genshin translations.

Perhaps you are opting to go this way because in Genshin `Hold CD` is in the format of `CD (hold)`?

Reply:
Yea I'm a bit anal when it comes to strings. Can revert this if you think it's unnecessary translation work. I also thought about just modifying the English translation as it is generated but didn't really explore it